ui/src/common/utils.js

```diff
@@ -224,3 +224,16 @@ export function abbreviateNumber(number) {
   const numberOfFractionDigits = number < 10000 ? 1 : 0;
   return numberAbbreviator.abbreviate(number, numberOfFractionDigits);
 }
+
+const HTML_TAG_REGEXP = /(<([^>]+)>)/ig;
+const MAX_DESCRIPTION_LENGTH = 160
+export function makeCompliantMetaDescription(description = '') {
```

Comment:
oh we need that for job description?

Reply:
well for anything... we have quite relaxed schema for all records.